encoding/text/marshal_test.go

```diff
@@ -59,8 +59,8 @@ func TestEncode(t *testing.T) {
 		{0x81fdbfdc91779421, `(map = [])`, `(
     map = []
 )`},
-		{0x8e85252144f61858, `(data = "Hi\xde\xad\xbe\xef\xca\xfe")`, `(
-    data = "Hi\xde\xad\xbe\xef\xca\xfe"
+		{0x8e85252144f61858, `(data = "Hi\xde\xad\xbe\xef\xca\xfe\"\'\\")`, `(
```

Comment:
I'm okay with merging this optimistically, but could you just give me a bit of context about what's going on here?  The change seems very simple, but I'm not familiar with this part of the code base, so I just want to do my due diligence. 🙂

Reply:
I assume this is just expanding the test, to cover the characters relevant to this patch.